app/views/hiring_staff/vacancies/application_details/show.html.haml

```diff
@@ -35,7 +35,8 @@
         label: { text: t('helpers.fieldset.application_details_form.application_link'), size: 's' },
         hint_text: t('helpers.hint.application_details_form.application_link')
 
-      = render 'hiring_staff/vacancies/vacancy_form_partials/submit', f: f
+      = render 'hiring_staff/vacancies/vacancy_form_partials/continue_or_update_submit', f: f
```

Comment:
I prefer the submit buttons being a single partial, but that might just be me

Reply:
Not possible because some pages have the two buttons and some have just the one.